ckanext/fluent/validators.py

```diff
@@ -1,15 +1,18 @@
 import json
 import re
+import six
 
-from ckan.plugins.toolkit import missing, _, get_validator, Invalid
-from pylons import config
+from ckan.plugins.toolkit import missing, get_validator, Invalid, check_ckan_version
+from ckan.common import config, _
```

Comment:
install `ckantoolkit` module and import `config` and `_` from it - there were no `config` inside `ckan.common` before 2.6